tests/model_registry/utils.py

```diff
@@ -163,6 +163,32 @@ def get_model_registry_db_label_dict(db_resource_name: str) -> dict[str, str]:
     }
 
 
+def get_model_registry_service(admin_client: DynamicClient, namespace: str) -> Service:
+    """
+    Get the Model Registry service.
+
+    Retrieves the Model Registry service from the specified namespace.
+    Args:
+        admin_client (DynamicClient): OCP Client to use.
+        namespace (str): Namespace where the Model Registry service is located.
+
+    Returns:
+        Service: The Model Registry service.
+
+    Raises:
+        RuntimeError: if no Model Registry service is found.
+    """
+
+    services = list(
+        Service.get(
+            dyn_client=admin_client, namespace=namespace, label_selector="app=model-registry,component=model-registry"
+        )
+    )
+    if not services:
+        raise RuntimeError(f"Model Registry service not found in namespace {namespace}")
+    return services[0]
```

Comment:
Can't you use https://github.com/opendatahub-io/opendatahub-tests/pull/283/files#diff-1d1c4a8accca671d431801d22056cfc7deee5b4bb6ff6f1f6c684a8fc17cae62L21 ?

Reply:
probably yes, let me try out

Reply:
removed function